packages/app/screens/group/tabs/AboutScreen.js

```diff
@@ -1,8 +1,123 @@
-import React from 'react';
-import { View } from 'react-native';
+import React from "react";
+import { StyleSheet, Text, View, Linking } from "react-native";
+import FontAwesome5 from "@expo/vector-icons/FontAwesome5";
+import Screen from "@app/components/Screen";
+import Ionicons from "@expo/vector-icons/Ionicons";
 
-function AboutScreen(){
-    return(<View></View>);
+const BLUE = "#3498DB";
+const GRAY = "#2C2C2C";
+
+function AboutScreen() {
+  return (
+    <Screen style={styles.container}>
+      <View style={styles.subContainer}>
+        <Text style={styles.title}>Description</Text>
+        <Text style={styles.description}>
+          Hey everyone, this is so fun! HOORAY!!! Hey everyone, this is so fun!
+          HOORAY!!! Hey everyone, this is so fun! HOORAY!!!
+        </Text>
+      </View>
+
+      <View style={styles.subContainer}>
+        <Text style={styles.title}>Links</Text>
+        <View style={styles.mediaContainer}>
+          <FontAwesome5
+            name="discord"
+            style={styles.mediaIconStyle}
+            size={20}
+          />
+          <Text
+            style={styles.url}
+            numberOfLines={1}
+            ellipsizeMode="tail"
+            onPress={() => Linking.openURL("https://discord.com")}>
+            https://discord.com
+          </Text>
+        </View>
+        <View style={styles.mediaContainer}>
+          <FontAwesome5
+            name="instagram"
+            style={styles.mediaIconStyle}
+            size={20}
+          />
+          <Text
+            style={styles.url}
+            numberOfLines={1}
+            ellipsizeMode="tail"
+            onPress={() => Linking.openURL("https://instagram.com")}>
+            https://instagram.com
```

Comment:
When getting media links from the guild context, you can get a random number of links. (i.e. no links, just a twitter link, or multiple links like twitter/x, instagram, discord, linkedin, and so on).

I would suggest using some kind of list or mapping these links or however you prefer. It may be a little complicated to display icons as well but you could possibly display icons in accordance to the link's string, although I'd say that can be an optional extra feature.